components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -350,3 +359,46 @@ func assertIssuerIsReachable(host string) error {
 	}
 	return nil
 }
+
+func assertIssuerProvidesDiscovery(ctx context.Context, issuer string) error {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
Intentionally allowing insecure?

Reply:
Don't recall the reason for this. Admittedly, I borrowed that from some other reachability check. I don't see value to have it relaxed. 
Corrected ✔️